README.md

```diff
@@ -8,7 +8,7 @@ There are two main categories of samples: **Camera** and **Applications**. The s
 
 - **Camera**
   - **Basic**
-    - **Capture** ([**hdev**][hdev-capture-url]) - This example shows how to acquire a 3D color point cloud from the camera and use it to generate a HALCON ObjectModel3D which is then visualized.
+    - **Capture** ([**hdev**][hdev-capture-url], [**cpp-Zivid**][cpp-captureViaZivid-url], [**cpp-GenICam**][cpp-captureViaGenICam-url]) - This example shows how to acquire a 3D color point cloud from the camera and use it to generate a HALCON ObjectModel3D which is then visualized.
```

Comment:
Does this commit belong here? Are you missing a rebase?

Reply:
that's a local proposal.. was discussing with Christian. Shouldn't have been there

Reply:
So remove or make a different commit?

Reply:
I'll fixup